src/Test/MakerTestDetails.php

```diff
@@ -70,6 +66,8 @@ public function addExtraDependencies(string ...$packages): self
 
     public function setRequiredPhpVersion(int $version): self
     {
+        @trigger_deprecation('symfony/maker-bundle', 'v1.44.0', 'setRequiredPhpVersion() is no longer used and will be removed in a future version.');
```

Comment:
Are we sure we won't need this again? :)

Reply:
Nope. But _we_ are not using it within the code base at the moment. Because the class isn't internal - I figured we'd throw the deprecation. And if/when we need to use this (PHP `9`ish) - we can either remove the deprecation or add a replacement `internal` method that gives us a bit more freedom. 

I _think_ we are going with a similar strategy w/ the `PHPCompatUtil` - we don't have a need for now (probably wont until PHP 9ish). So when we introduced the util in non-internal classes, we added a deprecation that "hey, you need to supply this util.. it'll be mandatory in the next version..." Now that we _don't_ need it in say MakeWhatever::class, I changed the wording on the deprecation "hey, you dont need to supply this..."

That strategy kinda sucks, but I can't think of a better way w/ non internal classes. Any ideas?

Reply:
works for me 👍